dandi/cli/cmd_organize.py

```diff
@@ -34,6 +36,18 @@
     default="auto",
     show_default=True,
 )
+@click.option(
+    "--rewrite-external-files",
+    type=click.Choice(["copy", "move"]),
+    default=None,
+    help="using this options will rewrite the 'external_file' argument of the "
+    "ImageSeries in the nwb file. The value written will correspond to the"
+    " new location of the video files after being organized. This option will "
+    "work only if --files-mode is copy/move since the original nwb file is "
+    "modified Supply one of 'copy' or 'move' as the value. Copy will copy the "
+    "video file in the organized dandiset "
+    "(Copy option is not recommended for large video files)",
+)
```

Comment:
@Saksham20 

It seems that these options are already handled by "--files-mode" above.

Reply:
(`--files-mode` , `--rewrite-external-files` ) can be any of permutation of (`copy`, `move`)